C++/Logger/conversion/ConvertibleTo.h

```diff
@@ -6,8 +6,11 @@ namespace Toolbox::Log::Conversion
 
 #include "../metaprogramming/value_type_is.h"
 
-template <typename ResultType>
-class ConvertibleTo : public value_type_is <ResultType>
```

Comment:
On mobile so it is hard to review but doesn't your value_type_is have support for variatic types?